### ISSUE TITLE
Adjust presentation of testimonials

### ISSUE THREAD

**Issue by peterfremlin** [peterfremlin/Disability-Debrief#30]
Currently:

<img width="669" alt="Screenshot 2024-03-29 at 18 27 43" src="https://github.com/peterfremlin/Disability-Debrief/assets/106767665/5174254c-767e-4d05-b8f9-2fa6dd5347f2">

- [x] too much whitespace above and below the testimonial
- [x] both quotation and person needs to be centered
- [x] if you have a way to avoid quotation hanging over by one line (or whether I should choose text accordingly) - in this case would look better if text wrapped earlier
- [x] the idea with testimonials was that they would disappear and a new one show in its place

That said, I am feeling that the site needs a redesign and I wonder when to do that rather than just tweaking

**Comment by peterfremlin** [comment]
Enjoying this a lot - nice transition. Will check access and in meantime my one comment is to slightly reduce whitespace above and below. (e..g to be the same as the whitespace between "sign-up" and the signup box)

**Comment by peterfremlin** [comment]
Also: 
- while the quotation in the testimonial is wrapping well, the attribution doesn't wrap. In longer attributions (name + organization) it goes beyond the width of screen
- the text of the attribution looks a slightly different font from others on page. Would make sense to match the text used in the articles pagination for the excerpt?

**Comment by cathysarisky** [comment]
Yes, we need to do better on accessibility.  Here are two sets of examples and guidelines:
https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-2-tablist/
https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-1-prev-next/

Obviously, in your case it's just text, not an image, but the implementation needs to be similar.  So we need the ability to advance between slides with keyboard or mouse.  And if there's autoplay, there needs to be a play/stop button.  That's going to clutter up the minimalist layout a bit.  Perhaps we want /only/ the prev/next buttons, no autoplay?

**Comment by peterfremlin** [comment]
Thanks Cathy. 

I would like to keep the autoplay I think an using aria-live="off" or aria-hidden="true" to just make the changes unobtrusive to a screen-reader user? (Setting those stops announcing the changes?)

Perhaps this could be combined with unobtrusive previous/next buttons - clicking those would stop autoplay and let someone navigate them on their own time. That's also useful for anyone who needs it at a different speed.

**Comment by cathysarisky** [comment]
Announcing stopped (at least with the chrome screenreader extension - will need validation by better readers)

Let's do a better navigation soon - I don't love what's there currently, from an accessibility standpoint, but at least it isn't spammy now.

**Comment by peterfremlin** [comment]
Áine (my screen-reader user who checks this) says that now testimonials aren't visible at all on the homepage.

The ideal behaviour for a screenreader user (at the moment) is to see the first one and not the following ones.

In terms of the one to come up by default please use:

“A wonderful resource.”
Haben Girma, disability rights lawyer 

I think this will be good for other users as well - if this testimonial is directly loaded onto the page then there isn't a jump from testimonials being invisible to being loaded.

**Comment by cathysarisky** [comment]
I've updated the aria settings, which will hopefully let the screen reader see (but not get spammed).  My thanks to Aine for checking!

If you'll please put the testimonial you want to show first in the first position, I'll adjust the js to match that.

**Comment by peterfremlin** [comment]
Thanks, I've just adjusted the one that I want to first position (it isn't the one that is loaded first for me without a screenreader)

The second part of what I'm saying is that currently there's a visual (and a bit annoying) jump on the page loading. It loads without testimonials and then it all shifts when they come in a moment later. I was wondering if hard coding the one from Haben would stop that happening to the user.

**Comment by cathysarisky** [comment]
I'm experimenting with a different carousel option... lmk play with that a bit.  And yes, we don't want jumping!

**Comment by peterfremlin** [comment]
closing here and moving to next milestone
